src/schema/v2/artwork/index.ts

```diff
@@ -1094,31 +1097,57 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return partnerLoader(partner.id).catch(() => null)
         },
       },
+      userPartnerOffersConnection: {
+        type: PartnerOfferConnectionType,
+        args: pageable({
+          includeExpired: {
+            type: GraphQLBoolean,
+          },
+          page: { type: GraphQLInt },
+          size: { type: GraphQLInt },
+          sort: { type: PartnerOfferSortsType },
+        }),
+        resolve: async (artwork, args, { mePartnerOffersLoader }) => {
+          if (!mePartnerOffersLoader) return null
+
+          const paginationArgs = convertConnectionArgsToGravityArgs(args)
+          const { page, size, offset } = paginationArgs
+
+          const gravityArgs = {
+            artwork_id: artwork.id,
+            total_count: true,
+            page,
+            size,
+          }
+
+          if (args.sort) {
+            gravityArgs["sort"] = args.sort
+          }
+          if (args.includeExpired) {
+            gravityArgs["include_expired"] = args.includeExpired
```

Comment:
This hasn't been implemented yet in Gravity right?

Reply:
It has now - artsy/gravtiy#17374